modules/logging-release-notes-5-6-20.adoc

```diff
@@ -0,0 +1,93 @@
+// module included in /logging/logging-5-6-release-notes
+:_mod-docs-content-type: REFERENCE
+[id="logging-release-notes-5-6-20_{context}"]
+= Logging 5.6.20
+This release includes link:https://access.redhat.com/errata/RHBA-2024:3740[OpenShift Logging Bug Fix 5.6.20]
+
+[id="logging-release-notes-5-6-20-bug-fixes"]
+== Bug fixes
+
+* Before this update, there was a delay in restarting Ingesters when configuring `LokiStack`, because the {loki-op} sets the write-ahead log `replay_memory_ceiling` to zero bytes for the `1x.demo` size. With this update, the minimum value used for the `replay_memory_ceiling` has been increased to avoid delays. (link:https://issues.redhat.com/browse/LOG-5617[LOG-5617])
```

Comment:
I just want to make sure Ingesters should be capitalized. When I search the OCP docs, I see it both ways. Like here is lowercase:
- https://docs.openshift.com/container-platform/4.15/observability/logging/troubleshooting/log-forwarding-troubleshooting.html

In distributed tracing though, its upper https://docs.openshift.com/container-platform/4.15/observability/distr_tracing/distr_tracing_arch/distr-tracing-architecture.html

Reply:
This issue has been resolved in Slack with @libander